db/migrate/20160203195736_remove_docker_image_schema.rb

```diff
@@ -0,0 +1,39 @@
+class RemoveDockerImageSchema < ActiveRecord::Migration
```

Comment:
@bbuckingham I misread "malleable" as "ordering not guaranteed" vs "we pick the ordering", I understand now. :eyeglasses: